packages/app-content-pages/src/screens/Publications/components/Publication/Publication.js

```diff
@@ -8,43 +8,56 @@ const StyledBox = styled(Box)`
   overflow: hidden;
 `
 
-const Placeholder = (
-  <Box
-    align='center'
-    justify='center'
-    background='brand'
-    height='100%'
-    width='100%'
-  >
-    <ZooniverseLogo size='50%' />
-  </Box>
-)
+const Placeholder = ({ id }) => {
+  return (
+    <Box
+      align='center'
+      justify='center'
+      background='brand'
+      height='100%'
+      width='100%'
+    >
+      <ZooniverseLogo size='50%' id={id} />
+    </Box>
+  )
+}
 
-function Publication (props) {
-  const { authors, avatarSrc, className, title, url, year } = props
+function Publication({
+  authors = '',
+  avatarSrc = '',
+  className = '',
+  title = '',
+  url = '',
+  year = ''
+}) {
   const displayString = [title, authors, year].filter(v => v).join(', ')
```

Comment:
What is the filter doing here?

Reply:
Honestly I'm not sure 🤔

Reply:
The filter helps in cases where one of the variables in array [title, authors, year] is missing. For example, if all three of those are missing the `displayString` is just two commas.

Reply:
In other words, we don't want to add commas to missing authors or year

Reply:
```suggestion
  const displayString = [title, authors, year].filter(v => !!v).join(', ')
```

Converting `v` to a boolean might clarify that. I’m not sure.